dev/com.ibm.ws.install/src/com/ibm/ws/install/internal/InstallKernelMap.java

```diff
@@ -961,24 +993,34 @@ public Collection<String> singleFileResolve() {
                         if (repoResrc.getRepositoryConnection() instanceof DirectoryRepositoryConnection) {
                             featuresResolved.add(repoResrc.getId());
                         } else {
-                            if (repoResrc.getMavenCoordinates() == null) { //if user specified ESA file path
+                            String featurePath = repoResrc.getMavenCoordinates();
+                            if (featurePath == null) { //if user specified ESA file path
                                 if (repoResrc instanceof EsaResourceImpl) {
                                     String name = ((EsaResourceImpl) repoResrc).getShortName() == null ? repoResrc.getName() : ((EsaResourceImpl) repoResrc).getShortName();
                                     for (String k : shortNameMap.keySet()) {
                                         if (shortNameMap.get(k).equals(name)) {
                                             featuresResolved.add(k);
+                                            featurePath = k;
```

Comment:
do we need to add featurePath to featuresResolved?

Reply:
I added on the line above `featurePath = k;`

Reply:
my bad i glanced right over that.